builtin/provisioners/chef/resource_provisioner.go

```diff
@@ -173,6 +175,9 @@ func (r *ResourceProvisioner) Validate(c *terraform.ResourceConfig) (ws []string
 	if p.ServerURL == "" {
 		es = append(es, fmt.Errorf("Key not found: server_url"))
 	}
+        if p.SecretKeyPath == "" {
+                es = append(es, fmt.Errorf("Key not found: secret_key_path"))
+        }
```

Comment:
This will make the `SecretKeyPath` mandatory, while it should be optional right?